IMC_WebDev/FinalBlog.md

```diff
@@ -0,0 +1,224 @@
+## Building a Mini City: HTML and CSS Portion
+## Purpose
+HTML and CSS are extremely important parts of learning about how websites work. Every website uses HTML to display content such as text or images. Being able to grasp a few concepts of  HTML and CSS can help a lot when it comes to learning more intricate subjects in computer science, so it's important to get some experience with it!
+
+This tutorial will give you a walkthrough on how certain functions of HTML work and why they are important, all while learning how to build a cool mini-city!
+
+## Setup
+Open any HTML/CSS editor. [This browser-based editor is good to use](https://www.w3schools.com/tryit/tryit.asp?filename=tryhtml_default).
+Follow along with this tutorial, and you will be one step closer to building your first mini-city!
+
+## Walkthrough
+
+This tutorial will go over the HTML and CSS portion of the mini-city. You can think of HTML as the foundation of the entire project. If we work to build a strong foundation, our mini-city will look fantastic!
+
+Our first step when using HTML is to establish a “!DOCTYPE html” function. This simply means that all of our code below this will be rendered as an HTML document.
+So, your first line should look like this: 
+```html
+<!DOCTYPE html>
+```
+
+Got it? Great!
+
+Next, we have to actually create our HTML tag. Tags are essentially commands that tell our editor what type of HTML we are writing. This just means that we are beginning to write our HTML code, but first we must actually tell our computer that we will be writing in HTML. To do this, simply type &lt;html&gt; on your second line. Your code should look like this so far: 
+```html
+<!DOCTYPE html>
+<html>
+```
+
+Now that we have established the fact that we will be writing in HTML, it’s time to get into the fun stuff. We start by writing &lt;head&gt; in our third line of code. This tells our editor that the information for our header will be stored here.
+
+After we write our &lt;head&gt; tag, we know that we should give our project a title. Let’s title our project “My Mini City.” This is how your code should look now: 
```

Comment:
For future reference, usually punctuation goes inside quotation marks. Such as "My Mini City." or "My Mini City," and...